pydactyl/api/locations.py

```diff
@@ -31,33 +31,33 @@ def get_location_info(self, location_id, includes=None, params=None):
             includes=includes, params=params)
         return response
 
-    def create_location(self, shortcode, description):
```

Comment:
Don't change the name of the parameters, this isn't necessary and it breaks anyone who is specifying these as named parameters.